src/Raven.Server/ServerWide/Commands/DeleteServerWideBackupConfigurationCommand.cs

```diff
@@ -12,11 +13,10 @@ protected DeleteServerWideBackupConfigurationCommand()
             // for deserialization
         }
 
+        [Obsolete]
         public DeleteServerWideBackupConfigurationCommand(string configurationName, string uniqueRequestId) : base(uniqueRequestId)
         {
-            Name = ClusterStateMachine.ServerWideBackupConfigurationsKey;
-            Value = configurationName;
-        }
+            throw new InvalidOperationException($"This command is obsolete, please use `{nameof(DeleteServerWideTaskCommand)}` instead");        }
```

Comment:
1. Better to do 'Obsolete(error: true)`
2. Formatting
3. Assuming that there is no flow that uses this ctor.

Reply:
done